<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Название сорта</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Степень обжарки</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string comment="1">Светлая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="2">Средняя</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="3">Темнее средней</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="4">Темная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="5">Очень темная</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Молотый/в зёрнах</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string comment="True">Молотый</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="False">В зёрнах</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Описание вкуса</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Объём упаковки</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Внести изменения</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Добавить/изменить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Изменить</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить кофе</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
